README.md

```diff
@@ -1,3 +1,7 @@
+# :warning::warning: DEPRECATED :warning::warning:
+
+This project, and it's related TestArmada projects, will no longer be supported. No further work from the owners will be done, and no PRs will be reviewed.
```

Comment:
## ⛏

Please remove the comma after `done` in the second sentence as it is not needed.

![image](https://user-images.githubusercontent.com/4923649/104785274-f9795780-573e-11eb-9837-ba56882358bc.png)